stdlib/private/SwiftPrivateLibcExtras/SwiftPrivateLibcExtras.swift

```diff
@@ -97,12 +97,21 @@ public func _stdlib_select(
         let readAddr = readfds.baseAddress
         let writeAddr = writefds.baseAddress
         let errorAddr = errorfds.baseAddress
+#if CYGWIN
+        func asFdSetPtr(
+          _ p: UnsafeMutablePointer<UInt>?
+        ) -> UnsafeMutablePointer<_types_fd_set>? {
```

Comment:
Probably because we are dropping the aliasing in the clang import.  Can you please file a SR and send it my way with a tiny test case (something like creating a function which uses `fd_set`)?